src/pages/view/TitleBar/ActionsBar.js

```diff
@@ -226,6 +234,15 @@ const ViewActions = ({
         </DropdownButton>
     )
 
+    const updateStarredFilter = () => {
+        const isStarred = isFilterStarred(starredFilter)
+        const value = isStarred ? '' : STARRED_STATE
+        setStarredFilter(value)
+        apiPostStarredDashboard(value)
+    }
```

Comment:
There is no spinner feedback on the action (confirm with @adrianq if we want it)

Reply:
@eperedo I wrongly said to @tokland we don't need a spinner but now that I am testing the app I think a spinner would improve the user experience. Is it too much work to add the dashboard spinner on button click?

Reply:
I'm adding the spinner on the button, what do you think?

![image](https://github.com/user-attachments/assets/7a468da2-0759-4a74-a42d-1ab75c571d6c)

![image](https://github.com/user-attachments/assets/5d0a8893-38d5-481a-8d0b-676f744eebe5)

Reply:
@eperedo It looks good to me. I will share it with the client. As mentioned in the card, the idea is to create a PR for Oslo. Having this in mind, do you think this is a standard solution in the dashboard app? In some cases, I can see they use a semi-transparant overlay...

Reply:
I've checked the Edit -> "Save changes" button, they have no feedback, so not sure what they will say about this. In Maintenance they have a change in the button, but no spinner. Capture, which seems a more approach, has this (while saving the attributes):

![image](https://github.com/user-attachments/assets/5fb91962-abbb-4593-9f7a-7d114ada91a3)